### DISCUSSION TITLE
Rider diff tool not used - it switched to VisualStudioCode

### DISCUSSION THREAD

**Discussion by ronheywood** [VerifyTests/Verify#1080] [Q&A] [answered]
Since I started with Verify as a tool in Rider, it's launched the Rider diff tool, which I've been quite happy with.

Today, after I worked in an old project using the Approvals package, somethings got twisted up and now my "VerifyTests" solution is using Visual Studio Code as the diff tool

The other solution - using the Approvals package is still happily launching diffs in the Rider diff tool...

I tried setting the diff tool order in a SetUp method:
```
    [SetUp]
    public void Setup()
    {

        DiffTools.UseOrder(DiffTool.Rider,DiffTool.VisualStudioCode);
    }
```    
But it still launches Visual Studio Code.

I'm confused about what I might have done to make this change, and how to undo it?

**Comment by SimonCropp** [comment]
what version of diffengine are you using? and where is the rider exe installed?

**Reply by ronheywood** [reply]
I'm on Verify.Nunit 22.5.0 (because I'm still using Nunit 3 for now)
DiffEngine  was Implicitly installed 13.0.0

Upgrading DiffEngine to 13.0.1 didn't change the behavior. 

Rider is in "C:\Program Files\JetBrains\JetBrains Rider 2022.1.2"

Let me try a quick Hello World project and see if it is using Rider for that...

**Reply by ronheywood** [reply]
Yeah a brand new empty test project:
```
[Test]
public async Task Test1()
{
      await Verifier.Verify("Hello World");
}
```
Also launches VSCode

I can't see any environment variables that might affect the behavior.
If I uninstall VSCode it launches Visual Studio

**Reply by SimonCropp** [reply, marked as answer]
can u try  DiffEngine 13.0.2

**Reply by ronheywood** [reply]
Success! - Thank you :)

**Reply by SimonCropp** [reply]
i will update Very and approvaltests tomorrow. then u can remove the direct ref to diffengine
